plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectCsvRecordReader.java

```diff
@@ -91,4 +92,11 @@ public SelectObjectContentRequest buildSelectObjectRequest(Properties schema, St
 
         return selectObjectRequest;
     }
+
+    @Override
+    protected String getFieldDelimiter(Properties schema)
+    {
+        // Use the field delimiter only if it is specified in the schema. If not, use null (defaults to ',' in S3 Select).
```

Comment:
What does it mean that null defaults to a comma?

Reply:
I updated the description in the latest commit. Basically, when you send the field delimiter as `null` on the `SelectObjectContent` request to S3 Select, the S3 Select service uses comma as the field delimiter. Since this class only deals with CSV file formats, it looks to be a reasonable default to me.